src/components/wallet/Wallet.js

```diff
@@ -130,6 +163,9 @@ export function Wallet() {
     const hideExploreApps = localStorage.getItem('hideExploreApps')
     const linkdropAmount = localStorage.getItem('linkdropAmount')
     const linkdropModal = linkdropAmount && showLinkdropModal !== false;
+    const { tokens } = useSelector(({ tokens }) => tokens)
+    const { actionStatus } = useSelector(({ status }) => status)
+    const tokensLoader = actionsPending(['TOKENS/LIKELY_CONTRACTS/GET', 'TOKENS/TOKENS_DETAILS/GET_METADATA', 'TOKENS/TOKENS_DETAILS/GET_BALANCE_OF']) || !balance?.total
```

Comment:
feel like it would be nice to just be able to pass `TOKENS/` prefix here

Reply:
added `actionsPendingByPrefix` function